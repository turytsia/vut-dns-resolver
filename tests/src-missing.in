-r -t www.fit.vutbr.cz