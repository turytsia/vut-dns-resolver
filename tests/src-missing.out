Error: Source address is missing for the options -s
